style.md

```diff
@@ -151,6 +152,65 @@ An interface is two fields:
 If you want interface methods to modify the underlying data, you must use a
 pointer.
 
+### Verify Interface Compliance
+
+For types that are known or required to implement specific interfaces, add
+checks to verify interface compliance at compile time.
+
+<table>
+<thead><tr><th>Bad</th><th>Good</th></tr></thead>
+<tbody>
+<tr><td>
+
+```go
+type Handler struct{ /* ... */ }
+
+
+
```

Comment:
nit: I think we can condense this whitespace

Reply:
The intent was for it to align with the addition of `var _ ...` on the right side of the table.

![image](https://user-images.githubusercontent.com/41730/74695041-5d01aa80-51a7-11ea-9e91-438720b52396.png)

I can get rid of it if the extraneous whitespace isn't worth it.

Reply:
ah, I was only looking at this in diff form, not in rendered form. makes sense.